### PR TITLE
Adds getPreviewFromContent() method for using link-preview-js with pre-fetched response object.

### PR DESCRIPTION
**Pull request by wuori**
Was really wanting to use this plugin within my Cordova (mobile) apps. 

**What does this "fix"?**
The nature of Cordova causes a lot of CORS issues when accessing outside domains from JS used within Cordova apps. In many cases, we use a plugin (e.g. [Cordova Http Plugin](https://www.npmjs.com/package/cordova-plugin-advanced-http) to access resources outside of the app's whitelisted domains.

**How does this fix work?**
Creates a new method named `getPreviewFromContent` in which you pass a pre-fetched Response object vs. the text/url you would otherwise pass in `getLinkPreview`.

Simply put: this just bypasses the `fetch` called by `getLinkPreview`, as the Response object is already provided.

**More Information on this PR**
-  Passes existing tests
- No new tests written
- Tested with a few domains, including Google.com

I'm in a bit of a hurry and would otherwise write more tests for this. I just wanted to get this in front of some eyes so if the idea itself is going to be rejected I don't have to waste my time :) 

Thank you for the plugin, it does everything I was looking for!

### CODE REVIEW

**Review by ospfranco — APPROVED** [review]

**Comment by ospfranco** [comment]
Can you update the README a bit more, I think it is mentioned in the beginning there is a single method.

Cheers!

**Comment by ospfranco** [comment]
Also please specify on the README the shape of the object it is expected to be parsed, I'm sure a lot of people will try to pass every kind of response and will lead to a lot of confusion

**Comment by wuori** [comment]
> Also please specify on the README the shape of the object it is expected to be parsed, I'm sure a lot of people will try to pass every kind of response and will lead to a lot of confusion

Pushed some updates, let me know if you'd like it to be more encompassing. Thanks!

**Comment by ospfranco** [comment]
I copied your branch and cleaned up a bit, closing this

https://github.com/ospfranco/link-preview-js/pull/75
